pom.xml

```diff
@@ -74,6 +74,7 @@
         <groupId>org.jenkins-ci.tools</groupId>
         <artifactId>maven-hpi-plugin</artifactId>
         <configuration>
+          <compatibleSinceVersion>1.15</compatibleSinceVersion>
```

Comment:
Is this true? If I upgrade to 1.15 I can no longer downgrade to 1.14 without restoring the configuration from a backup?